debugview/src/debug/kotlin/com/revenuecat/purchases/debugview/InternalDebugRevenueCatBottomSheet.kt

```diff
@@ -16,6 +16,7 @@ internal fun InternalDebugRevenueCatBottomSheet(
     onPurchaseErrored: (PurchasesTransactionException) -> Unit,
     isVisible: Boolean = false,
     onDismissCallback: (() -> Unit)? = null,
+    viewModel: DebugRevenueCatViewModel? = null,
```

Comment:
It think this could clarify that passing `nil` will create a default view model.

Reply:
I will add a comment 👍